package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "clean": "rimraf build && mkdir build",
     "lint": "eslint .",
-    "test": "babel-node test/index.js",
+    "test": "tape -r babel/register test/index.js",
```

Comment:
To enable debugging. There is that tool called `iron-node`. It work in the following way - in your command line you replace `node` with `iron-node`.

When I prepend `iron-node` to `babel-node` the debugging doesn't work.
```
iron-node babel-node test/index.js
```
The received errors are quite difficult to solve.

Whereas, if I prepend `iron-node` to the `tape` command line it just works:
```
iron-node tape -r babel/register test/index.js
```

That was the only reason really. I hope that's fine.